tests/component-plugin/src/main/resources/META-INF/mule-test.xsd

```diff
@@ -301,4 +301,40 @@
         </xsd:complexContent>
     </xsd:complexType>
 
+    <xsd:element name="set-payload" type="setPayloadTransformerType" substitutionGroup="mule:abstract-message-processor">
```

Comment:
Since `set-payload` is going to become unavailable here soon and `transform` won't be available in CE I thought we should have a test version to make tests (and their migration) easier. When the `set-payload` task is done the Processor will just need to be moved to the test plugin.